hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/ServerCrashProcedure.java

```diff
@@ -482,13 +482,18 @@ private void assignRegions(MasterProcedureEnv env, List<RegionInfo> regions) thr
           regionNode.getProcedure().serverCrashed(env, regionNode, getServerName());
           continue;
         }
-        if (env.getMasterServices().getTableStateManager().isTableState(regionNode.getTable(),
-          TableState.State.DISABLING, TableState.State.DISABLED)) {
+        if (env.getMasterServices().getTableStateManager()
+          .isTableState(regionNode.getTable(), TableState.State.DISABLING)) {
           env.getAssignmentManager().regionClosedAbnormally(regionNode);
-          LOG.info("{} found table disabled for region {}, set it state to ABNORMALLY_CLOSED.",
+          LOG.info("{} found table disabling for region {}, set it state to ABNORMALLY_CLOSED.",
             this, regionNode);
           continue;
         }
+        if (env.getMasterServices().getTableStateManager()
+          .isTableState(regionNode.getTable(), TableState.State.DISABLED)) {
+          LOG.warn("{} found table disabled for region {}.", this, regionNode);
```

Comment:
`this` would print `server`, `splitWal` and `meta` correct? or it will print `pid`, `ppid`, `state` etc additionally?

Reply:
Seems like it will print everything, is it good to log:
```
LOG.warn("Found table disabled for region {}, procDetails: {}", regionNode, this);
```

Reply:
done, change it.